docs/source/commandline/commandline-all.rst

```diff
@@ -648,125 +733,238 @@ Core State/Playback Options
+    path separator on any target platform (even Windows); the string ``%g``
+    represents the driver name of the current system.
+
+    The default is ``%g``, which creates a separate folder for each system.
 
-	Two simple substitutions are provided: the / character represents the path separator on any target platform (even Windows); the string ``%g`` represents the driver name of the current system.
+    In addition to the above, for drivers using different media, like carts or
+    floppy disks, you can also use the ``%d_[media]`` indicator. Replace
+    ``[media]`` with the media switch you want to use.
 
-	The default is ``%g``, which creates a separate folder for each system.
+    A few examples:
 
-	In addition to the above, for drivers using different media, like carts or floppy disks, you can also use the ``%d_[media]`` indicator. Replace ``[media]`` with the media switch you want to use.
+    If you use **mame robby -statename foo/%g** save states will be stored
+    inside ``sta\\foo\\robby\\``.
 
-	A few examples: if you use **mame robby -statename foo/%g** save states will be stored inside ``sta\\foo\\robby\\``; if you use **mame nes -cart robby -statename %g/%d_cart** save states will be stored inside ``sta\\nes\\robby\\``; if you use **mame c64 -flop1 robby -statename %g/%d_flop1** save states will be stored inside 'sta\\c64\\robby\\'.
+    If you use **mame nes -cart robby -statename %g/%d_cart** save states will
+    be stored inside ``sta\\nes\\robby\\``.
+
+    If you use **mame c64 -flop1 robby -statename %g/%d_flop1** save states will
+    be stored inside 'sta\\c64\\robby\\'.
 
 .. _mame-commandline-noburnin:
 
 **-[no]burnin**
 
-	Tracks brightness of the screen during play and at the end of emulation generates a PNG that can be used to simulate burn-in effects on other systems. The resulting PNG is created such that the least used-areas of the screen are fully white (since burned-in areas are darker, all other areas of the screen must be lightened a touch).
+    Tracks brightness of the screen during play and at the end of emulation
+    generates a PNG that can be used to simulate burn-in effects on other
+    systems.  The resulting PNG is created such that the least used-areas of
+    the screen are fully white (since burned-in areas are darker, all other
+    areas of the screen must be lightened a touch).
+
+    The intention is that this PNG can be loaded via an artwork file with a low
+    alpha (e.g, 0.1-0.2 seems to work well) and blended over the entire screen.
 
-	The intention is that this PNG can be loaded via an artwork file with a low alpha (e.g, 0.1-0.2 seems to work well) and blended over the entire screen. The PNG files are saved in the snap directory under the ``systemname/burnin-<screen.name>.png``. The default is OFF (**-noburnin**).
+    The PNG files are saved in the snap directory under the
+    ``systemname/burnin-<screen.name>.png``.
```

Comment:
Since "systemname" isn't literal it should be in angle brackets like "screen.name" is.

Reply:
Corrected.